examples/run.html

```diff
@@ -32,6 +32,20 @@
 
       function onAddRemoteStream(event) {
         html5VideoElement.srcObject = event.streams[0];
+
+        let intervalToken = setInterval(
+          async () => {
+            let stats = await webrtcPeerConnection.getStats();
+            if (websocketConnection.readyState == 3) {
```

Comment:
curiosity, which state is 3?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/readyState